tests/providers/google/cloud/hooks/test_bigquery.py

```diff
@@ -203,7 +203,10 @@ def test_cancel_queries(self, mock_client, mock_poll_job_complete):
         self.hook.running_job_id = running_job_id
         self.hook.cancel_query()
 
-        calls = [mock.call(job_id=running_job_id), mock.call(running_job_id)]
+        calls = [
```

Comment:
@uranusjr Thx for your comment. The two test cases test `cancel_query` which calls `cancel_job`. They also check the calls invoking `poll_job_complete`. So I think it is good enough. During test, I need to set an environment variable `AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT=google-cloud-platform://` because the `GoogleBaseHook` [calls](https://github.com/apache/airflow/blob/main/airflow/providers/google/common/hooks/base_google.py#L231) `get_connection` in its constructor. I think we should move that function call within a member function. But it is out of the scope of this PR.

One concern is that `cancel_query` is being deprecated. When we fully deprecate it, we shall convert some of the test cases to cover `cancel_job`.